db/migrate/20220503230849_add_reviewed_by_at_and_by_to_contributions.rb

```diff
@@ -0,0 +1,6 @@
+class AddReviewedByAtAndByToContributions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :contributions, :reviewed_by, :string
```

Comment:
why is this a string and not a relation?

Reply:
@HeitorMC This column should points out to an User